nix-tools/static/project.nix

```diff
@@ -0,0 +1,93 @@
+inputs: pkgs:
```

Comment:
This file comes from [this snippet](https://github.com/input-output-hk/haskell-nix-example/blob/9b130e2c0128a55ac02820d3b297c07a6ddade10/flake.nix#L425-L499) in haskell-nix-example